_sass/components/_eiti-slider.scss

```diff
@@ -1,12 +1,10 @@
 eiti-slider {
-  background: #fff;
-  cursor: pointer;
+  $slider-background: darken($mid-blue, 30%);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets